examples/pytorch/speech-recognition/run_speech_recognition_seq2seq.py

```diff
@@ -95,7 +95,10 @@ class ModelArguments:
         },
     )
     freeze_feature_encoder: bool = field(
-        default=True, metadata={"help": "Whether to freeze the feature encoder layers of the model."}
+        default=False, metadata={"help": "Whether to freeze the feature encoder layers of the model."}
```

Comment:
```suggestion
        default=True, metadata={"help": "Whether to freeze the feature encoder layers of the model."}
```

We should keep backwards compatibility here no?

Reply:
I guess `freeze_feature_encoder` is not implemented in `WhisperPreTrainedModel`, so it will break.

Reply:
Absolutely @wade3han! For Whisper we'll just pass the argument `freeze_feature_encoder=False` whenever we launch the script to override this default.